src/electron/views/automated-checks/components/automated-checks-view.tsx

```diff
@@ -1,10 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-
 import * as React from 'react';
 
-export class AutomatedChecksView extends React.Component {
-    public render(): JSX.Element {
-        return <>automated checks</>;
-    }
-}
+import { NamedFC } from 'common/react/named-fc';
+import { CommandBar, CommandBarProps } from './command-bar';
+import { HeaderSection } from './header-section';
+
+export type AutomatedChecksViewProps = CommandBarProps;
```

Comment:
We usually write props explicitly and leave the implicit pass-through for deps, no? Should we do that here?

Reply:
Thats a good point but since this is not the final version of this component and we don't have any prop/dep directly use by `AutomatedChecksView`, I'll resolve the comment on a future PR when we add the title bar component and we actually wire up this component